### ISSUE TITLE
command.stdout.on() gets empty string when responses contain Chinses characters（Windows）

### ISSUE THREAD

**Issue by brsmsg** [tauri-apps/tauri#2673]
### **Describe the bug**
In our App, we use a java program to write JSON string into a standard IO stream, and we use listener to track the stream through ```command.stdout.on("data",  (data) => { console.log(data) })```. 
There are some Chinese characters in the response data and we lost the response in this API (the terminal shows an empty strings).
Such a problem only exists when responses contain some Chinese characters on Windows. On macOS and Ubuntu, it works well.
We also tested our API directly using Node.js on Windows and we got the Chinese content correctly. 

### **To Reproduce**
use ```command.stdout.on("data",  (data) => { console.log(data) })``` and data string has some Chinses characters.

### **Expected behavior**
```command.stdout.on()``` can get Chinese characters correctly. If there are some encoding problems, at least the api can get us some Chinese Garbled instead of an empty string.

### **Platform and Versions (required):**

```
Operating System - Windows version 10.0.18363.535  X64

Node.js environment
Node.js - 14.16.0
@tauri-apps/cli - 1.0.0-beta.10
@tauri-apps/api - 1.0.0-beta.8

Global packages
npm - 6.14.11
pnpm - 6.0.2

Rust environment
rustc - 1.54.0
cargo - 1.54.0

App directory structure
/.git
/.vscode
/dist
/node_modules
/public
/release
/src
/src-tauri

App
tauri.rs - 1.0.0-beta.8
build-type - bundle
CSP - default-src blob: data: filesystem: ws: wss: asset: http: https: tauri: 'unsafe-eval' 'unsafe-inline' 'self' img-src: 'self'
distDir - ../dist
```

**Comment by YNVoon** [comment]
Is there any alternative workaround? I am having this issue when my Windows language is in Chinese.

**Comment by lucasfernog** [comment]
Unfortunately I tried to fix this a year ago but it was super complicated :( Windows attacks again.

**Comment by lucasfernog** [comment]
Fixed by #4772
